src/interface/RHInterface.py

```diff
@@ -92,7 +93,7 @@ def unpack_rssi(node, data):
 
 
 class RHInterface(BaseHardwareInterface):
-    def __init__(self):
+    def __init__(self,**kwargs):
```

Comment:
I'm not sure whether it is a good idea to use kwargs instead of an explicit param. kwargs would make sense to me if we are explicitly passing config as individual args. As we are just passing a single config object, it is not obvious to me what the benefit is?

Reply:
I set it up to just pass the entire config dictionary even though only some of it is used. You are correct, there is no benefit, so I will pass it explicitly.